cl/scrapers/transformer_extractor_utils.py

```diff
@@ -0,0 +1,47 @@
+import json
+from tempfile import NamedTemporaryFile
+
+import requests
+from django.conf import settings
+from django.core import serializers
+
+
+def serialize_audio_object(af):
+    """Convert serialize audio object into json for processing.
+
+    :param af: Audio File object
+    :return: Modified serailized audio file for processing in BTE
+    """
+    af_dict = json.loads(serializers.serialize("json", [af]))[0]["fields"]
+    docket_dict = json.loads(serializers.serialize("json", [af.docket]))[0][
+        "fields"
+    ]
+    court_dict = json.loads(serializers.serialize("json", [af.docket.court]))[
+        0
+    ]["fields"]
+    af_dict["docket"] = docket_dict
+    af_dict["docket"]["court"] = court_dict
+    with NamedTemporaryFile(suffix=".json") as tmp:
+        with open(tmp.name, "w") as json_data:
+            json.dump(af_dict, json_data)
+        with open(tmp.name, "rb") as file:
+            af = file.read()
+    return af
+
+
+def convert_and_clean_audio(audio_obj):
```

Comment:
A few things here:

1. The convert URL feels like it could go in the settings file. Maybe make a dict of URLs there, so you can look them up with something like `BTE_URLS["convert_audio"]`. Glance around the settings file to see other examples of this kind of thing.

1. This also highlights that `BTE_URL` should be renamed `BTE_HOST`.

1. Your docstring is complete (yay!), but, alas, it's wrong. You're not returning a processed audio file, you're returning a Response object. This confused me for a bit because up above you have:

        response = convert_and_clean_audio(af).json()

    And I was like, "How can he call `json()` on an Audio file?" Also, why is your variable named `response` if it's holding the json, not the response? Gotta clean this up.

Reply:
Oh, and one more thing. I like the tidiness of your code below, but you should separate reading the file from doing a request so that the errors are clearer and easier to catch. That is, it's better to do:

```python3
with ... some_file:
    contents = some_file.read()
requests.post(...some stuff, timeout=xyz)
```

Reply:
I've unwrapped file reading, and made the settings changes with bte urls and bte hosts.